include/xparrow/xparrow_version.hpp

```diff
@@ -5,11 +5,11 @@
 *                                                                          *
 * The full license is in the file LICENSE, distributed with this software. *
 ***************************************************************************/
-#ifndef XPARROW_VERSION_HPP
-#define XPARROW_VERSION_HPP
+#pragma once
 
-#define XPARROW_VERSION_MAJOR 0
-#define XPARROW_VERSION_MINOR 0
-#define XPARROW_VERSION_PATCH 1
-
-#endif
+namespace xparrow
+{
+    constexpr int XPARROW_VERSION_MAJOR = 0;
+    constexpr int XPARROW_VERSION_MINOR = 0;
+    constexpr int XPARROW_VERSION_PATCH = 1;
```

Comment:
You need to update the [CMakeLists.txt](https://github.com/xtensor-stack/xparrow/blob/main/CMakeLists.txt#L18) accordingly

Reply:
Should be fixed now, but I didnt rename yet